apps/prairielearn/src/components/QuestionScore.html.ts

```diff
@@ -249,12 +249,33 @@ export function ExamQuestionScore({
 
 export function ExamQuestionStatus({
   instance_question,
+  assessment_question,
 }: {
   instance_question: InstanceQuestion & {
     allow_grade_left_ms?: number;
     allow_grade_interval?: string;
   };
+  assessment_question: Pick<AssessmentQuestion, 'max_auto_points' | 'max_manual_points'>;
 }) {
+  // Special case: if this is a manually graded question in the "saved" state,
+  // we want to differentiate it from saved auto-graded questions which can
+  // be graded immediately. We'll use a green badge so that student can drive
+  // towards all status badges being green.
+  //
+  // TODO: can we safely look at the assessment question for exams? What about
+  // the guarantee that an Exam-type assessment won't change after it's created?
+  if (
+    instance_question.status === 'saved' &&
+    !assessment_question.max_auto_points &&
+    assessment_question.max_manual_points
+  ) {
+    return html`
+      <span class="align-middle">
+        <span class="badge badge-success">saved for manual grading</span>
```

Comment:
This caused overflow for me:
<img width="372" alt="image" src="https://github.com/user-attachments/assets/f1e381f7-fff2-48d8-98ec-39dbdad7e77d">

Reply:
What do you think about changing "Submission status" to just "Status"? If we did this, I'd make the same change on the assessment instance page too.

Reply:
Discussed with @mwest1066, we'll make this change.

Reply:
I opened https://github.com/PrairieLearn/PrairieLearn/pull/10831 to make this change.

Reply:
🎉